src/Raven.Server/Web/Studio/LicenseHandler.cs

```diff
@@ -1,7 +1,10 @@
 ﻿using System;
 using System.Net;
+using System.Net.Http;
+using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+using Newtonsoft.Json;
```

Comment:
Please remove usings that are not needed

Reply:
Removed